tests/treactiveSigil.nim

```diff
@@ -0,0 +1,38 @@
+import sigils/reactive
+
+import unittest
+import std/sequtils
+
+suite "reactive examples":
+
+  test "reactive wrapper":
+
+    let
+      x = reactive(5)
+      y = computed[int]():
+        2 * x{}
+
+    check x.val == 5
+    check y.val == 0
```

Comment:
As mentioned earlier, I'd more expect an eager evaluation, so it makes sense for me that y.val here should be 2*x => 10 at that point in time.

Reply:
Indeed it should. Good catch!

Reply:
Hey, doing that actually makes use of the first run of the block and makes the code look cleaner. Noice